### PR TITLE
Last cucumber tests

### PR DESCRIPTION
**Pull request by ulitol97**
Cucumber tests implemented for:
- Entity queries its incidents
- Person checks the status of his incidents

Covering #49

### CODE REVIEW

**Comment by ulitol97** [comment]
IMO everything is fine, I'll let you review it just in case before merging.

**Comment by alejgh** [comment]
Yeah, everything looks fine. You can merge new tests without reviews if you want, don't worry about that.

Btw, thanks for fixing the typo in `a_person_with_usrname_and_password`, I saw it once and I forgot to change it 😆

**Review by alejgh — APPROVED** [review]

**Comment by ulitol97** [comment]
Perfect! And you're welcome :P

**Pull request merged**
